projects/wave-core/src/lib/layers/layer-list/layer-list.component.ts

```diff
@@ -90,6 +78,18 @@ export class LayerListComponent implements OnDestroy {
         this.projectService.toggleSymbology(layer);
     }
 
+    getLayerSymbologyStream<T extends AbstractSymbology>(layer: Layer<T>): Observable<T> {
+        return this.projectService.getLayerChangesStream(layer).pipe(
+            startWith(layer),
+            filter((lc) => !!lc.symbology),
+            map((c) => layer.symbology)
```

Comment:
The variable `c` is redundant and can be removed (empty function)

Reply:
hab ich gelöscht